owned-manifests/clusterapi-manager-controllers.yaml

```diff
@@ -55,18 +47,8 @@ spec:
           limits:
             cpu: 100m
             memory: 30Mi
-      {{- if eq .Provider "aws" }}
-      - name: aws-machine-controller
-        image: {{ .Images.ClusterAPIControllerAWS }}
-      {{end}}
-      {{- if eq .Provider "openstack" }}
-      - name: openstack-machine-controller
-        image: {{ .Images.ClusterAPIControllerOpenStack }}
-      {{end}}
-      {{- if eq .Provider "libvirt" }}
-      - name: libvirt-machine-controller
-        image: {{ .Images.ClusterAPIControllerLibvirt }} 
-      {{- end}}
+      - name: machine-controller
+        image: {{ .Controller }}
```

Comment:
How does it now chose the correct one?

Reply:
discrimination happens in go https://github.com/openshift/machine-api-operator/pull/106/files#diff-b77facdf2f36a2cc873ec9ea3fae6b00R138